Source/WebCore/testing/Internals.idl

```diff
@@ -567,7 +567,7 @@ typedef (FetchRequest or FetchResponse) FetchObject;
     unsigned long imagePendingDecodePromisesCountForTesting(HTMLImageElement element);
     undefined setClearDecoderAfterAsyncFrameRequestForTesting(HTMLImageElement element, boolean enabled);
     unsigned long imageDecodeCount(HTMLImageElement element);
-    unsigned long pdfDocumentCachingCount(HTMLImageElement element);
+    unsigned long viewportCreateCount(HTMLImageElement element);
```

Comment:
this needs "imageViewport" in the name, because "viewport" can mean lots of things (visual viewport, layout viewport etc).

Reply:
Fixed.